apps/site/pages/docs/building-sections/overrides/use-cases/add-an-image-to-your-alert-component.mdx

```diff
@@ -0,0 +1,136 @@
+    extends Omit<
+        React.ComponentProps<typeof AlertSection>,
+        "icon"
+    > {
+    src: string;
+    }
+   ```
+
+   - This interface defines the props accepted by the `AlertWithImage` component.
+   - It inherits all props from the native Alert component, excluding the icon prop as we intend to replace it with an image.
+   - It adds a new prop named `src`, which specifies the image source URL.
+
+3. Create the `AlertWithImage` component:
+
+   ```tsx
+    import { AlertSection, getOverriddenSection } from "@faststore/core";
+    import { Image_unstable as Image } from "@faststore/core/experimental";
+    import { useMemo } from "react";
+    interface AlertWithImageProps
+    extends Omit<
+        React.ComponentProps<typeof AlertSection>,
+        "icon"
+    > {
+    src: string;
+    }
+
+    export default function AlertWithImage(props: AlertWithImageProps) {
+    const { src, ...otherProps } = props;
+
+    /**
+    * This is an alert override that contains an Image as an Icon.
+    *
+    * Changes the CMS schema to add the src option and remove the icon option.
+    * Overrides the Icon option and passes down the other props
+    *
+    * It's memoized to avoid being re-created every time the AlertWithImage component is re-rendered
+    */
+    const OverriddenAlert = useMemo(
+        () =>
+        getOverriddenSection({
+            Section: AlertSection,
+            components: {
+            Icon: { Component: () => <Image src={props.src} alt="custom icon" width={24} height={24} /> },
+            },
+        }),
+        []
+    );
+
+    return <OverriddenAlert {...otherProps} icon="noop" />;
+    }
+   ```
+
+   - This function receives the `AlertWithImageProps`.
+   - It destructures the `src` prop containing the image source and remaining props using the spread operator (`...otherProps`).
+   - The core functionality lies within the `useMemo` hook.
+       - It defines the `OverriddenAlert` constant using `getOverriddenSection`.
+       - The override options specify:
+           - **`Section`:** The section to be overridden (Alert).
+           - **`components`:** An object containing the override for the Icon component.
+               - Inside the Icon override:
+                   - **`Component`:** A function returning an `Image` component. This displays the image specified by the `src` prop.
+                   - **`width`** and **`height`:** Optional props to set the size of the image.
+       - The use of `useMemo` ensures this override is only created once, improving performance.
+   - The function returns the `OverriddenAlert` component, spreading the `otherProps` and setting the `icon` prop to `noop` (effectively disabling the default icon behavior).
+
+4. Add the section to the Headless CMS by following the instructions available in [Sending components to the Headless CMS](/docs/building-sections/overrides/sending-components-to-the-headless-cms).
+For the schema it was used the following:
```

Comment:
I think we need to link, where the people can find the original schema. Which is [here](https://faststore-site-git-docs-override-version-2-faststore.vercel.app/docs/building-sections/overrides/list-of-native-sections#faststore-native-sections)